stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -400,6 +415,22 @@ extension Unsafe${Mutable}RawBufferPointer {
     return baseAddress!.load(fromByteOffset: offset, as: T.self)
   }
 
+  // The public version of this doesn't use this method because this unsafely
+  // unwraps the base address. The public store bytes uses
```

Comment:
The "The public store bytes uses _unsafelyUnwrappedUnchecked, should the public ones too..?" phrase is a bit confusing. Is that correct?

Reply:
ah yeah it's a little confusing. I meant that in the public implementation of `storeBytes` it uses `._unsafelyUnwrappedUnchecked` whereas the public implementation of `load` uses the bang operator on the base address. Let me reword it a little.